Dropdown/Dropdown.module.less

```diff
@@ -57,6 +61,8 @@
 			border-bottom-color: @agate-dropdown-border-color;
 			border-bottom-style: @agate-dropdown-border-style;
 			border-bottom-width: @agate-dropdown-border-width;
+			padding-top: @agate-dropdown-list-padding-top;
+			padding-bottom: @agate-dropdown-list-padding-bottom;
```

Comment:
We'd probably be better served here by using a single variable containing shorthand values.